domains/measurements/DataHubFacadeClients/DataHubFacadeClients.csproj

```diff
@@ -13,7 +13,7 @@
 		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
 	</PropertyGroup>
 	<ItemGroup>
-	  <PackageReference Include="Google.Protobuf" Version="3.27.0" />
+	  <PackageReference Include="Google.Protobuf" Version="3.27.3" />
```

Comment:
Don't know if this might break something. Since it's not updated with GRPC aswell.